src/main/java/com/github/terma/jenkins/githubprcoveragestatus/CompareCoverageAction.java

```diff
@@ -35,16 +36,29 @@
 import hudson.tasks.Publisher;
 import hudson.tasks.Recorder;
 import jenkins.tasks.SimpleBuildStep;
+import org.kohsuke.stapler.DataBoundSetter;
 
 @SuppressWarnings({"unused", "WeakerAccess"})
 public class CompareCoverageAction extends Recorder implements SimpleBuildStep {
 
     private static final long serialVersionUID = 1L;
+    private String sonarLogin;
+    private String sonarPassword;
```

Comment:
I think better place to move that setting close to other to global configuration ```Configuration```, I don't think people will be use different accounts per build.